sdk/cosmosdb/cosmos/package.json

```diff
@@ -108,6 +108,10 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
     "@azure/logger": "^1.0.0",
+    "@azure/core-tracing": "^1.0.0",
```

Comment:
This should be a regular dependency, since it's loaded in the cosmos sources and not just the tests or tools.